.github/workflows/generic-amd64.yml

```diff
@@ -5,7 +5,8 @@
 # to the inputs and the defaults.
 
 # TODO: We need a system to keep these inputs aligned across all device repos
-name: Generic x86_64 (GPT)
+# Name workflow after machine
```

Comment:
Use `${{ github.workflow }}` in the search rather than depending on a specific syntax for names. What if we need to build for the same machine in different modes?

Reply:
We can remove this change to the workflow name now